integration-tests/smoke/pom.xml

```diff
@@ -99,31 +99,6 @@
         </executions>
       </plugin>
 
-      <plugin>
```

Comment:
I think this is required because of Greenmail, a library that simulates a mail server, so that I can test the invitation process.